CHANGES.md

```diff
@@ -24,6 +24,17 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.2.2
+
+Update to `raiwidgets` package.
```

Comment:
Repeating this in every release seems a bit superfluous. Otherwise this looks fine, but @imatiach-msft should comment on the bug fixes since they were his changes.